content/root/best-practices-guide.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Best Practices Guide
+section: root
+index: 30
+jump_to:
+  Contents:
+    - Ably 101
+    - Using Ably like a pro
+---
+
+h2(#title). Ably Best Practices Guide
+
+In order to avail the best advantage of our platform, be sure to follow this best practices guide while building your applications using Ably.
+
+h3(#ably-101). Ably 101
+
+Ably Realtime is a cloud-based platform that allows you to easily integrate realtime functionality into your applications. Just plug and play.
+
+h4(#pubsub). Publish and Subscribe
+
+Publish data to a named "channel":/realtime/channels-messages on the Ably Realtime platform, and everyone subscribed to that same channel will receive it.
+
+h4(#notifications). Notify
+
+You can use "Push Notifications":/realtime/push to inform your users about the cool new features you are building in your application.
+
+h4(#presence). Be present
+
+Go online and let others in your application know this by using "Presence":/realtime/presence. You can also see who else is present on a channel even when you yourself are NOT present.
+
+h4(#rest-or-realtime). Remain connected, or not
+
+With the availability of both the "Realtime":/realtime and "REST":/rest libraries, choose what suits your application's needs best and get started.
+
+h4(#auth). Choose the right auth scheme
+
+Use "Basic Authentication":/general/authentication#basic-authentication or "Token Authentication":/general/authentication#token-authentication as well as ensure security by "encrypting your messages.":/realtime/encryption
```

Comment:
Authentication and encryption are not related concepts.  Why have you grouped them together?
Also, security is not "encryption", security is a far broader concept that may encompass many features including auth.
You have not explained here why one should use Basic auth or Token auth, but instead have linked to the documentaiton on eahc.  The point of this document is surely to help people understand best practice, yet this document does not do that. It's just mirroring our documentation links.  Best practice would for example state that typically in app, device or browser clients, we recommend you use Token Authentication because the device is untrusted and you can dynamically assign capabilities to each client from your servers, and a realtime client library should be used to maintain a connenction to Ably and subscribe to realtime updates.  Typically on your servers, we most often see people using API keys as this is a simpler authentication scheme when used on private servers you can trust, and REST libraries are often simpler to use because servers often have no need to maintain a connection to Ably as they are publishing messages, or issuing tokens for example.

The above is "best practice".  What you have provided is a list of features with links to our docs, which they have already.

Reply:
I understand your concern, @mattheworiordan. However, this is explained in point 5 within 'Using Ably like a pro' section. This section was really meant for the readers to know broadly what all features Ably offers (so they can go understand it from the docs if they hadn't known about it before) whereas the actual best practices/ recommendations are provided in the next section. This is the same reason why I provided links within the previous points too. However, I've updated these points so they make sense when read as a standalone document too.